docs/libcurl/curl_getdate.md

```diff
@@ -65,7 +65,7 @@ UTC. Supported formats include: -1200, MST, +0100.
 ## day of the week items
 
 Specifies a day of the week. Days of the week may be spelled out in full
-(using English): `Sunday', `Monday', etc or they may be abbreviated to their
+(using English): `Sunday`, `Monday`, etc or they may be abbreviated to their
```

Comment:
This isn't supposed to be marked up as code, but is using the older style quoting of `quotedvalue'. This being parsed as markdown makes it render weird in Github so we should just make them 'quotedvalue' instead.